src/engine/ruleset/integrations/windows/rules/defense-evasion-deleting-websvr-access-logs.yml

```diff
@@ -0,0 +1,39 @@
+name: rule/defense-evasion-deleting-websvr-access-logs/0
+
+metadata:
```

Comment:
Isn't it necessary to add the compatibility section?

Reply:
the rules are not compatible